### DISCUSSION TITLE
Other ways to pick notes

### DISCUSSION THREAD

**Discussion by oyvindsolstad** [javalent/obsidian-leaflet#58] [Ideas]
I LOVE the plugin. There are so many things I'm going to use it for. So thank you! Well done, great job!

**Would you consider making more options to which notes get used to grab front matter from?** 

As of now, we can use a folder, a document or tags. 

In the Dataview plugin, I can also choose links from a note, or to a note. That would make it easier to use. 

One example. Map with a line (boat, bus, train, plane). And stops on the map. The way the plugin is now, I would either have to make a special tag for these stops, or put them in a special folder. I would like to say that the map should show all notes that link to a certain note. So if the note is called [[Paris-Nice line]], the map would simply display all notes that had locations that linked to that note. No need for tags or moving notes to a special folder. 

=) Øyvind

**Comment by valentine195** [comment]
This would be possible, but it would (like the tags option) require Dataview to be installed.

This makes me think, though - once the Dataview plugin API is complete, it would be nice to just have a dataview query option to load notes, if the plugin is installed. That way you could write any kind of query you like and then Dataview could handle finding the notes.

**Reply by oyvindsolstad** [reply]
Sounds great! Thanks!

**Reply by zhubib1234** [reply]
In the current version of leaflet, how does DataView interact with leaflet? Could you give me a more detailed introduction?

**Reply by valentine195** [reply]
Currently, the only interaction is: if you specify a `markerTag` in the code block, it will use Dataview's tag cache to search for notes with that tag that can be turned into markers.

**Reply by zhubib1234** [reply]
thanks！

**Reply by Moonbase59** [reply]
@oyvindsolstad: Very interesting use case! To be able to use the outgoing or incoming links of/to a note might turn out real useful.

@valentine195: You always put something on top! :grin: Personally, I’m always a little reluctant relying on another plugin if you yourself are one (more, possibly changing, APIs), but I can see your proposal would open an endless bunch of new possibilities.

**Comment by valentine195** [comment]
## 3.15.0 

- added two new parameters:

1. `linksTo`
2. `linksFrom`


Please see [Links](https://github.com/valentine195/obsidian-leaflet-plugin#links).

**Reply by oyvindsolstad** [reply]
Love it!

**Reply by valentine195** [reply]
Let me know if you run into any issues.

**Reply by oyvindsolstad** [reply]
It works perfectly! Screenshot with great locations for drone shots for trains in Norway (first location I added was [this](https://www.instagram.com/p/CP20C2ZKtHP/) location):

![Skjermbilde 2021-06-08 kl  13 06 49](https://user-images.githubusercontent.com/4631971/121174663-70340000-c85a-11eb-84ff-bee5f4c50450.png)

This makes it so much easier to organise locations, shots, stations etc in Obsidian! So BIG thanks from me!

**Reply by Moonbase59** [reply]
New great use cases to be found every day here! Love the idea of mapping best drone shot places.
